### DISCUSSION TITLE
Neptune 3 PETG Unable to get decent print

### DISCUSSION THREAD

**Discussion by RichBr1** [NARUTOfzr/Neptune_3#72] [Q&A]
Cannot get any decent prints out of the Neptune 3 using Cura latest and Neptune 3 profiles. for PETG. Stringing everywhere and terrible top layers. Have spent hours changing everything in Cura (temp, speed and many other settings) but nothing works,? 
PLA works a treat.
Anybody know of a magic setting to cure this.
Tried all temps from 210 to 245, Run at 40 , 50 and 60 mm/s
Fan from 20% to %0 %
Fills from 10 to 30%
Retraaction fro 2 to 6 mm
Brand new Eryone PETG White
Completely baffled

**Comment by Moody66** [comment]
I don't think you understand.  This is to report issues for the firmware.
The Facebook page is for help with prints and settings. The firmware has no
issues.

**Comment by RichBr1** [comment]
Apologies, it was not very clear what this site was for, will try elsewhere.Sent from Android deviceOn 4 Jul 2023 18:26, Moody66 ***@***.***> wrote:
I don't think you understand.  This is to report issues for the firmware.

The Facebook page is for help with prints and settings. The firmware has no

issues.

**Comment by Moody66** [comment]
Join the Facebook group. That is to help people for prints. We are all
there.  https://m.facebook.com/groups/192499235897759/
